simulators/portal/src/suites/beacon/sync.rs

```diff
@@ -0,0 +1,197 @@
+use crate::suites::beacon::bridge::service::BridgeService;
+use crate::suites::beacon::constants::{
+    BOOTNODES_ENVIRONMENT_VARIABLE, TRUSTED_BLOCK_ROOT_ENVIRONMENT_VARIABLE,
+};
+use crate::suites::environment::PortalNetwork;
+use ethportal_api::{BeaconNetworkApiClient, Discv5ApiClient};
+use hivesim::{dyn_async, Client, NClientTestSpec, Test};
+use std::collections::HashMap;
+use std::sync::Arc;
+use tokio::time::{sleep, timeout, Duration};
+
+dyn_async! {
+   pub async fn test_beacon_sync<'a> (test: &'a mut Test, _client: Option<Client>) {
+        // Get all available portal clients
+        let clients = test.sim.client_types().await;
+
```

Comment:
```suggestion
        let clients: Vec<ClientDefinition> = clients.into_iter().filter(|client| client.name != *TRIN_BRIDGE_CLIENT_TYPE).collect();

```

don't we need to filter trin_bridge?